src/context.rs

```diff
@@ -111,6 +135,11 @@ impl ContextBuilder {
         let cipher_list = std::ffi::CString::new(cipher_list)
             .or(Err(Error::fatal(wolfssl_sys::WOLFSSL_FAILURE)))?;
 
+        // SAFETY: [`wolfSSL_CTX_set_cipher_list`][0] ([also][1]) is documented to get the valid `ctx` pointer from wolfSSL_CTX_new() and
+        // `list` parameter which should be a null terminated C string pointer.
```

Comment:
More important than what WolfSSL requires is how our code meets that requirement, i.e. it's important to mention something like "which is guaranteed by the use of `std::ffi::CString::as_c_str()` here." wording.